apps/src/netsim/PubSubChannel.js

```diff
@@ -0,0 +1,64 @@
+/**
+ * @overview Wrapped pub/sub service channel APIs (like Pusher's Channel)
+ */
+/* jshint
+ funcscope: true,
+ newcap: true,
+ nonew: true,
+ shadow: false,
+ unused: true,
+
+ maxlen: 90,
+ maxparams: 3,
+ maxstatements: 200
+ */
+'use strict';
+
+/**
+ * Javascript interface for a publish/subscribe channel.
+ * @interface
+ */
+var PubSubChannel = exports;
+
+/**
+ * Subscribe to an event, so that the given callback is called when the
```

Comment:
Is this jsdoc in the wrong place?  I can't see any immediately following declaration that it applies to; the next thing is the jsdoc for the constructor.